doc_source/managing-kube-proxy.md

```diff
@@ -5,19 +5,21 @@
+| Kubernetes version | `1.22` | `1.21` | `1.20` | `1.19` |
+| --- | --- | --- | --- | --- | --- |
+| kube\-proxy \(default version\) | 1\.22\.11-eksbuild\.2 | 1\.21.\14-eksbuild\.2 | 1\.20\.15-eksbuild\.2 | 1\.19\.16-eksbuild\.2 |
+| kube\-proxy \(minimal\) | 1\.22\.11-minimal-eksbuild\.2 | 1\.21\.14-minimal-eksbuild\.2 | 1\.20\.15\-minimal\-eksbuild\.3 | 1\.19\.16\-minimal\-eksbuild\.3 |
 
-If you have a `1.18` or later cluster that you have not added the `kube-proxy` Amazon EKS add\-on to, you can add it using the procedure in [Adding the `kube-proxy` Amazon EKS add\-on](#adding-kube-proxy-eks-add-on)\. If you created your `1.18` or later cluster using the AWS Management Console after May 3, 2021, the `kube-proxy` Amazon EKS add\-on is already on your cluster\. If you created your `1.18` or later cluster using any other tool, and want to use the `kube-proxy` Amazon EKS add\-on, then you must add it to your cluster yourself\.
+If you have a `1.19` or later cluster that you have not added the `kube-proxy` Amazon EKS add\-on to, you can add it using the procedure in [Adding the `kube-proxy` Amazon EKS add\-on](#adding-kube-proxy-eks-add-on)\. If you created your `1.19` or later cluster using the AWS Management Console after May 3, 2021, the `kube-proxy` Amazon EKS add\-on is already on your cluster\. If you created your `1.19` or later cluster using any other tool, and want to use the `kube-proxy` Amazon EKS add\-on, then you must add it to your cluster yourself\.
 
-If you've added the `kube-proxy` Amazon EKS add\-on to your `1.18` or later cluster, you can manage it using the procedures in the [Updating the `kube-proxy` Amazon EKS add\-on](#updating-kube-proxy-eks-add-on) and [Removing the `kube-proxy` Amazon EKS add\-on](#removing-kube-proxy-eks-add-on) sections\. For more information about Amazon EKS add\-ons, see [Amazon EKS add\-ons](eks-add-ons.md)\.
+If you've added the `kube-proxy` Amazon EKS add\-on to your `1.19` or later cluster, you can manage it using the procedures in the [Updating the `kube-proxy` Amazon EKS add\-on](#updating-kube-proxy-eks-add-on) and [Removing the `kube-proxy` Amazon EKS add\-on](#removing-kube-proxy-eks-add-on) sections\. For more information about Amazon EKS add\-ons, see [Amazon EKS add\-ons](eks-add-ons.md)\.
 
 If you have not added the `kube-proxy` Amazon EKS add\-on, the `kube-proxy` self\-managed add\-on is still running on your cluster\. You can manually update the `kube-proxy` self\-managed add\-on using the procedure in [Updating the `kube-proxy` self\-managed add\-on](#updating-kube-proxy-add-on)\.
 
+For version compatiblity and skew policy, EKS adopts upsteam kubernetes [kube-proxy version skew policy](https://kubernetes.io/releases/version-skew-policy/#kube-proxy). That means, EKS version at minor version N can support upto N-2 minor versions of kube-proxy addon.
```

Comment:
upsteam --> upstream

Reply:
How about this minor rewording?  
"For version compatibility and skew policy, EKS adopts upstream kubernetes [version skew policy for kube-proxy](https://kubernetes.io/releases/version-skew-policy/#kube-proxy). So, EKS at minor version N can support upto N-2 minor versions of kube-proxy addon."

Reply:
Adopted with a minor change. Thank you.